### DISCUSSION TITLE
Export/Create collections in a single file and bulk download in different location in Lazer

### DISCUSSION THREAD

**Discussion by Mcproking** [ppy/osu#29380] [Ideas]
This feature can be beneficial for:

- Beatmap packs
- Tournament Map pools
- Player

Using a single collection file that can be opened in lazer and start the bulk download beatmaps for that collection.
After the beatmaps are downloaded, the client can also automatically create a collection folder in-game.
Players will be able to benefit from it by allowing them to export their collection into a single file which they can bring around.

Maybe the collection file can consist of the beatmapset IDs and beatmap IDs which the laser can read from it.

**Comment by Stoppedpuma** [comment]
See #13025
